src/open_r1/__init__.py

```diff
@@ -11,26 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-__version__ = "0.3.0.dev0"
-
-from .configs import DataArguments, DPOConfig, H4ArgumentParser, ModelArguments, SFTConfig
-from .data import apply_chat_template, get_datasets
-from .decontaminate import decontaminate_humaneval
-from .model_utils import get_checkpoint, get_kbit_device_map, get_quantization_config, get_tokenizer
-
-
-__all__ = [
```

Comment:
I think all this was copied over from H4, by @lewtun ?

Reply:
yes, and forgot to be removed when lewis refactored I think

Reply:
Yep, good catch!